app/controllers/contestants_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ContestantsController < ApplicationController
+    def index
+        @contestants = Contestant.all
+    end
+
+    def show
+        @contestant = Contestant.find(params[:id])
+    end
+
+    private
+
+    def contestant_params
+        params.permit(:id, :name, :age, :hometown, :years_of_experience, :project_id)
+    end
+end
```

Comment:
Is there a reason why all of your controllers are exactly the same with and `index`, `show`, and strong params? A lot of this implementation is not needed.